blocksync/blocksync_test.go

```diff
@@ -61,8 +60,8 @@ func TestSyncTaskInterval(t *testing.T) {
 
 func generateP2P() network.Overlay {
 	c := &config.Network{
-		Host: "127.0.0.1",
-		Port: 10001,
+		Host:                    "127.0.0.1",
```

Comment:
Are you using golang 1.11? Most of us are still on golang 1.10. From 1.10 to 1.11, `go fmt` changes the format behavior. So we prefer using the same version to avoid unnecessary code format change back and forth.

Reply:
Yes, let me pin golang to 1.10 on my machine